components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -2573,4 +2580,17 @@ private static String getSPQualifier(QueryParamDTO[] queryParamDTOs) {
         }
         return null;
     }
+
+    private static String resolveUrl(String defaultUrlContext, String urlFromConfig) throws URLBuilderException {
+
+        if (!IdentityTenantUtil.isTenantQualifiedUrlsEnabled() && StringUtils.isNotBlank(urlFromConfig)) {
+            if (log.isDebugEnabled()) {
+                log.debug("Resolved URL:" + urlFromConfig + " from file configuration for default url context:" +
```

Comment:
```suggestion
                log.debug("Resolved URL:" + urlFromConfig + " from file configuration for default url context: " +
```

Reply:
I don't think the space is required as the delimiter is there. Yet, it will bring bit more clarity